src/cmd/services/m3dbnode/config/config_test.go

```diff
@@ -206,6 +206,13 @@ db:
             size: 8192
             lowWatermark: 0.01
             highWatermark: 0.02
+        writeBatchPool:
+            initialBatchSize: 1024
```

Comment:
Should we use 128 to match the default write ingestion batch size?

Reply:
This is just a random config forwards/backwards test, but I can update it here